backend/de.metas.adempiere.adempiere/base/src/main/java-gen/de/metas/picking/model/I_PickingProfile_PickingJobLineConfig.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2025 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.picking.model;
+
+import org.adempiere.model.ModelColumn;
+
+/** Generated Interface for PickingProfile_PickingJobLineConfig
+ *  @author metasfresh (generated) 
+ */
+@SuppressWarnings("unused")
+public interface I_PickingProfile_PickingJobLineConfig 
```

Comment:
why creating this table for only one flag?
i think it would be harder to grasp for the admin/user than having this checkbox in header or even better, in the picking options.

Reply:
Well, all things must start somewhere, since this would be about setting **line** configs, I thought it's a good thing to have them separated. Ofc, if you disagree, I can put in the header.